csharp/src/Microsoft.ML.OnnxRuntime/SessionOptions.cs

```diff
@@ -37,7 +37,7 @@ public enum ExecutionMode
     public class SessionOptions : IDisposable
     {
         private IntPtr _nativePtr;
-        private static string[] cudaDelayLoadedLibs = { "cublas64_100.dll", "cudnn64_7.dll" };
+        private static string[] cudaDelayLoadedLibs = { "cublas64_10.dll", "cudnn64_7.dll", "curand64_10.dll" };
```

Comment:
add comment that it's for a specific CUDA version? 
btw, how does this not get exercised during CI testing? how come we didn't hit the dll not found exception thrown below?

Reply:
Sorry, I don't know. We need to investigate it.